docs/calicoctl/node.md

```diff
@@ -0,0 +1,211 @@
+                            "false" to run in the foreground.  When using
+                            libnetwork, this may not be set to "false".
+                            [default: true]
+  --log-dir=<LOG_DIR>       The directory for logs [default: /var/log/calico]
+  --ip=<IP>                 The local management address to use.
+  --ip6=<IP6>               The local IPv6 management address to use.
+  --as=<AS_NUM>             The default AS number for this node.
+  --ipv4                    Show IPv4 information only.
+  --ipv6                    Show IPv6 information only.
+  --kubernetes              Download and install the kubernetes plugin.
+  --rkt                     Download and install the rkt plugin.
+  --libnetwork              Use the libnetwork plugin.
+  --libnetwork-image=<LIBNETWORK_IMAGE_NAME>    Docker image to use for
+                            Calico's libnetwork driver.
+                            [default: calico/node-libnetwork:latest]
+
+```
+
+## calicoctl node commands
+
+
+### calicoctl node 
+
+This command starts a container using the `calico/node` Docker image. 
+
+It is required to run the `calicoctl node` command prior to configuring 
+endpoints to use Calico networking.  In order to run the command, the host must 
+be running Docker and must have have access to a running etcd instance.
+
+This command must be run as root and must be run on the specific Calico node 
+that you are configuring.
+
+Command syntax:
+
+```
+calicoctl node [--ip=<IP>] [--ip6=<IP6>] [--node-image=<DOCKER_IMAGE_NAME>] 
+    [--as=<AS_NUM>] [--log-dir=<LOG_DIR>] [--detach=<DETACH>] [--kubernetes] 
+    [--rkt] [(--libnetwork [--libnetwork-image=<LIBNETWORK_IMAGE_NAME>])]
+
+    <IP>: Unique IPv4 address associated with an interface on the host machine.
+    <IP6>: Unique IPv6 address associated with an interface on the host machine.
+    <DOCKER_IMAGE_NAME>: Desired calico/node Docker image to use.
+                         (default value depends on calicoctl binary version)
+    <AS_NUM>: Autonomous System number to use for BGP peering.
+              (default global AS number of 64511 is used if not specified)
+    <LOG_DIR>: Directory where Calico will store logs, if not default.
+               (default: /var/log/calico)
+    <DETACH>: Boolean to have calico/node run as detached (true) or in the foreground (false).
+              (default: true)
+    <LIBNETWORK_IMAGE_NAME>: Desired calico/node-libnetwork Docker image to use when 
+                             using the Docker libnetwork driver.
+
+    --kubernetes: Download and install the kubernetes plugin.
+    --rkt: Download and install the rkt plugin.
+    --libnetwork: Download and run the calico/node-libnetwork Docker image.
+```
+
+When running the `calicoctl node` command with the `--libnetwork` plugin, the 
+command starts a container using the `calico/node-libnetwork` Docker image in 
+addition to starting the `calico/node` Docker image.
+
```

Comment:
This one need a bit more love.

 - explain `--kubernetes`, `--rkt` as well
 - explain `--detach` --- e.g. use this option if you're adding Calico to an init system
 - explain what the IP addresses are used for e.g. identify src address for BGP peering and thus interface over which workload traffic will flow.